cargo-near/src/commands/new/mod.rs

```diff
@@ -133,6 +88,76 @@ impl NewContext {
     }
 }
 
+#[tracing::instrument(
+    target = "tracing_instrument",
+    name = "The process of executing",
+    skip_all
+)]
+fn execute_git_commands(project_dir: &std::path::Path) -> near_cli_rs::CliResult {
+    tracing::Span::current().pb_set_message("`git` commands ...");
+    tracing::info!(target: "near_teach_me", parent: &tracing::Span::none(), "Execution command: `git init`");
```

Comment:
This snippet also landed on main https://github.com/near/cargo-near/commit/fa934ed4a5cf1f985a0ba6b8c296566690b9de56

Reply:
done